django/views/debug.py

```diff
@@ -907,12 +884,14 @@ def default_urlconf(request):
 Installed Middleware:
 {{ settings.MIDDLEWARE_CLASSES|pprint }}
 
-{% if template_does_not_exist %}Template Loader Error:
-{% if loader_debug_info %}Django tried loading these templates, in this order:
-{% for loader in loader_debug_info %}Using loader {{ loader.loader }}:
-{% for t in loader.templates %}{{ t.name }} ({{ t.status }})
-{% endfor %}{% endfor %}
-{% else %}Django couldn't find any templates because your 'loaders' option is empty!
+{% if template_does_not_exist %}Template loader postmortem
```

Comment:
Sure. Do you mean indentation in the template code or indentation in the output? Whitespace gets output verbatim since this goes into a textarea, Some of the existing output displays poorly because of this. For instance, here's output from a ``TemplateSyntaxError``:

```
  1 : {% load badtags %}



   2 : 



   3 : {% block content %}



   4 :   Hello



   5 :   {{ a }}



   6 :    {{ coconuts }} 



   7 : {% endblock %}



   8 : 
```

Fixing that is possible but requires making the template code harder to read.

A couple thoughts:

1) If filters could be registered via the ``Engine``, I think a custom filter could be used here to collapse whitespace and get both a readable template and output.

2) Is there any reason not to reuse the ``TECHNICAL_500_TEXT_TEMPLATE`` for the copy/paste text box? That would make things a bit simpler, also.